composer-lock-diff

```diff
@@ -80,29 +81,44 @@ function version($pkg)
     return $version;
 }
 
+function sourceUrl($pkg) {
+    if (empty($pkg->source->url)) {
+        return '';
+    }
+
+    return str_replace('.git', '', $pkg->source->url);
+}
+
 function tableize($header, $data, $opts = array()) {
     if (empty($data)) return '';
 
     $opts = array_merge(array('capped' => true, 'joint' => '+'), $opts);
 
     if ($opts['no-links']) {
         array_walk($data, function(&$values) {
-            unset($values[2]);
+            unset($values[3]);
+            unset($values[4]);
         });
     } else {
         if (array_key_exists('url_formatter', $opts)) {
             $formatter = $opts['url_formatter'];
             array_walk($data, function(&$values) use ($formatter) {
-                $values[2] = call_user_func($formatter, $values[2], '...');
+                $values[0] = call_user_func($formatter, $values[4], $values[0]);
+                $values[3] = call_user_func($formatter, $values[2], '...');
```

Comment:
```suggestion
                $values[3] = call_user_func($formatter, $values[3], '...');
```

Looks like the compare URL in markdown isn't chosen correctly.

```
| Dev Changes                                                                     | From    | To      | Compare       |
|---------------------------------------------------------------------------------|---------|---------|---------------|
| [doctrine/instantiator](https://github.com/doctrine/instantiator)               | 1.0.5   | 1.1.0   | [...](1.1.0)  |
| [mockery/mockery](https://github.com/mockery/mockery)                           | 0.9.10  | 0.9.11  | [...](0.9.11) |
```

Reply:
Sorry, fixed it.

Reply:
Sorry, fixed it.